lib/services/network_manager.dart

```diff
@@ -76,9 +94,9 @@ Future<void> _onConnectivityChange(
   _networkAutomationLogger.finest(
       "Network Change: ${connections?.map((element) => element.toString()).join(", ") ?? "None (likely a manual function call)"}");
   connections ??= await Connectivity().checkConnectivity();
-  final [offlineModeActive, baseUrlChanged] = await Future.wait([
+  final [_, baseUrlChanged] = await Future.wait([
     _setOfflineMode(connections),
-    changeTargetUrl(),
+    changeTargetUrl()
```

Comment:
why did you remove the trailing comma? :)

Reply:
because I way playing with the idea of being able to toggle between ping and location based detection